### DISCUSSION TITLE
Support for prefabs / Unity object references ?

### DISCUSSION THREAD

**Discussion by lgarczyn** [cathei/BakingSheet#39] [Q&A]
So if I understand the project properly, including prefabs would require: 

* Make the prefabs addressable 

* Store the addressable path into the data sheet

* Load the prefab using a converter 

Is that a good approach? Do converters support async loads?

**Comment by cathei** [comment]
The workflow is correct. You’ll access asset thru `AddressablePath`, which supports async loading similar to `AssetReference`

https://github.com/cathei/BakingSheet/blob/master/UnityProject/Packages/com.cathei.bakingsheet/Runtime/Extensions/Addressables/AddressablePath.cs
